src/bidiMapper/scripts/eval.spec.ts

```diff
@@ -0,0 +1,162 @@
+import fs from 'fs/promises';
+import path from 'path';
+import * as chai from 'chai';
+import chaiExclude from 'chai-exclude';
+
+chai.use(chaiExclude);
+
+describe('Evaluator', function () {
+    let EVALUATOR: any;
+
+    // Get EVALUATOR.
+    before(async function () {
+        const eval_text = await fs.readFile(
+            path.join(__dirname, './eval.es'),
+            'utf8'
+        );
+
+        EVALUATOR = eval(eval_text);
+    });
+
+    function checkSerializeAndDeserialize(originalObject: any, expectedSerializedObj: any, excluding: string[] = []) {
```

Comment:
Consider using `unknown` instead of `any` where possible. I think it will work here. It's similar to `any` but forces you to type check the variable before you can operate on it: https://dmitripavlutin.com/typescript-unknown-vs-any/

Reply:
Good suggestion. I tried it out, but definition of `deepEqualExcludingEvery` uses `keyof T` as an ignored properties type. Using `unknown` makes properties of type `never[]`, which forces to use a cast in the call `excluding as any as never[]`, which looks a bit ugly for my taste.
```
 deepEqualExcludingEvery<T>(actual: T | T[], expected: T | T[], props: keyof T | (keyof T)[], message?: string): void;
```

Reply:
Sounds good, thanks for giving this a try.